docker/elasticsearch/README.md

```diff
@@ -0,0 +1,5 @@
+#### 2.0 Start Elasticsearch with docker
+If you familier with docker you can use the elasticsearch docker image with installed elasticsearch-vector-scoring plugin.
+Just build the image with 
+```docker build -t es-with-vector-scoring .```
```

Comment:
i think it'll be helpful to show the full output of both commands and state which directory to run them from